src/pretix/base/forms/widgets.py

```diff
@@ -60,6 +60,16 @@ def replace_arabic_numbers(inp):
     return inp.translate(table)
 
 
+def format_placeholders_help_text(placeholders, event=None):
+    phs = [
+        '<button type="button" class="content-placeholder" title="%s">{%s}</button>' % (v.render_sample(event) if event else v, k)
```

Comment:
I think the `title` should contain something like "Sample: " in front of the sample to explain what that piece of information is. (Also, I notice that not all of them have samples, e.g. meta parameters with no default have an empty sample.)

Reply:
Good catch with the meta_values! Regarding „Sample“ – good idea, although for some it is not a sample, e.g. when using `{event}` within an event’s settings. But generally prefixing with „Sample: “ is better. Pushed.